src/entities/Artifact.js

```diff
@@ -2,7 +2,8 @@ export default class Artifact {
   constructor(taskId, data, runId) {
     Object.assign(this, data);
     this.taskId = taskId;
-    this.isPublicLog = /^public\//.test(this.name);
+    this.isPublic = /^public\//.test(this.name);
+    this.isLog = /\/logs\//.test(this.name);
```

Comment:
I wonder if we should write down this convention somewhere?  Why not `/\.log$/`?

Or maybe this is just a kind of UI magic that we can plan to revisit later, and not commit to anything now?  I think the object service will let us query content-type, for example, so we could at that point default to the log viewer for anything `text/*`.

Reply:
> Why not `/\.log$/`?

So that something like https://tools.taskcluster.net/groups/b9fwUkb9TS-EI9YdwLw6_w/tasks/A73VbaXFRSCm5LRvKfB4Iw/runs/0/logs/public%2Flogs%2Fprecomplete.diff would still open in the logviewer.

> we could at that point default to the log viewer for anything text/*.

Good idea! We could keep `isLog` but have it return `true` when the content-type is `text/*`.